src/pages/workspaces/List.js

```diff
@@ -77,13 +77,15 @@ const styles = {
     margin: '1rem 4.5rem'
   },
   toolbarButtons: {
-    marginLeft: 'auto', display: 'flex',
-    backgroundColor: 'white', borderRadius: 3
+    borderRadius: 3, border: `1.45px solid ${colors.blue[0]}`,
```

Comment:
Even though the invision CSS can be helpful as a starting point, we should always apply our own judgements to it when translating it to code. Specifically:
* This border should just be 1px
* This box shadow is faint enough as to be almost non-existent, and none of our other buttons have one; we should exclude it
* The dimensions below should be rounded to reasonable numbers (and in the case of buttons, matched to the same height as the rest of the buttons)